xsd/netex_framework/netex_utility/netex_utility_types.xsd

```diff
@@ -453,21 +453,21 @@ Rail transport, Roads and Road transport
 	</xsd:element>
 	<xsd:complexType name="InfoLinkStructure">
 		<xsd:annotation>
-			<xsd:documentation>Type for Info LinK.</xsd:documentation>
+			<xsd:documentation>Type for Info Link.</xsd:documentation>
 		</xsd:annotation>
 		<xsd:simpleContent>
 			<xsd:extension base="xsd:anyURI">
 				<xsd:attribute name="typeOfInfoLink">
 					<xsd:annotation>
-						<xsd:documentation>Functional classifcation of infolink</xsd:documentation>
+						<xsd:documentation>Functional classifcation of an info link.</xsd:documentation>
 					</xsd:annotation>
 					<xsd:simpleType>
 						<xsd:list itemType="TypeOfInfolinkEnumeration"/>
```

Comment:
Uppercases and lowercases: I would prefer TypeOfInfo**L**inkEnumeration, and Info Link on line 456 aligned with lines 423 and 462.

Reply:
That is not a typo change anymore, but I get your point :-)